texts/epn_paper.md

```diff
@@ -0,0 +1,67 @@
+# Let's talk online
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home, hopefully safe and healthy.
+Your classes are now online, and probably you had to quickly figure out how run them at all.
```

Comment:
Your classes are now taught online, and you've probably had to quickly figure out how run them at all.